src/test/java/com/dabsquared/gitlabjenkins/publisher/TestUtility.java

```diff
@@ -131,7 +131,7 @@ static String formatNote(AbstractBuild build, String note) {
     static <P extends MergeRequestNotifier> P preparePublisher(P publisher, AbstractBuild build) {
         P spyPublisher = spy(publisher);
         MergeRequest mergeRequest =
-                new MergeRequest(MERGE_REQUEST_ID, MERGE_REQUEST_IID, "", "", "", PROJECT_ID, PROJECT_ID, "", "");
+                new MergeRequest(MERGE_REQUEST_ID, MERGE_REQUEST_IID, "", "", "", "", PROJECT_ID, PROJECT_ID, "", "");
```

Comment:
Could you please use a non-empty dummy string as the test merge commit SHA in the test? Otherwise how do we know if it works?

Reply:
> Could you please use a non-empty dummy string as the test merge commit SHA in the test? Otherwise how do we know if it works?

Hello, I have added this. Could you check?